store/sqlstore/user_store.go

```diff
@@ -232,17 +232,15 @@ func (us SqlUserStore) ResetLastPictureUpdate(userId string) *model.AppError {
 	return nil
 }
 
-func (us SqlUserStore) UpdateUpdateAt(userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		curTime := model.GetMillis()
-
-		if _, err := us.GetMaster().Exec("UPDATE Users SET UpdateAt = :Time WHERE Id = :UserId", map[string]interface{}{"Time": curTime, "UserId": userId}); err != nil {
-			result.Err = model.NewAppError("SqlUserStore.UpdateUpdateAt", "store.sql_user.update_update.app_error", nil, "user_id="+userId, http.StatusInternalServerError)
-			return
-		}
+func (us SqlUserStore) UpdateUpdateAt(userId string) (*model.User, *model.AppError) {
```

Comment:
The `UpdateUpdateAt` function should return and int64 (number of milliseconds of the current time when you update the `UpdateAt` field). Actually, looks like we don't use that anywhere, so probably is better to just return a `*model.AppError` directly.

Reply:
My fault, it is used in the tests, so we should re-write part of the tests to check it properly